libraries/AP_HAL/RCOutput.h

```diff
@@ -249,6 +249,11 @@ class AP_HAL::RCOutput {
      */
     virtual bool get_output_mode_banner(char banner_msg[], uint8_t banner_msg_len) const { return false; }
 
+    /*
+     * return mask of channels that must be disabled because they share a group with a digital channel
+     */
+    virtual uint16_t get_disabled_channels(uint16_t digital_mask) { return 0; }
```

Comment:
we might want to rename the "digital_mask" argument to servo_channel_mask or something similar.

to me, all our outputs are "digital".. they're not "analog" anyway which makes them digital doesn't it?

Reply:
The passed in mask are channels that are using dshot or LEDs and require DMA. Digital seems as good a name as any.